android/sdl_android/src/main/java/com/smartdevicelink/proxy/SdlProxyBase.java

```diff
@@ -1013,8 +1017,11 @@ public void handleQueueingError(String info, Exception ex) {
 				_outgoingProxyMessageDispatcher = null;
 			}
 			throw e;
-		} 

```

Comment:
Did the spacing change causing a delete and add?

Reply:
Adding new code right after the line (new line character) causes the editor to make the line delete/add.